css/css-multicol/columnfill-auto-widows-002.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+
+  <meta charset="UTF-8">
+
+  <title>CSS Multi-column Test: column-fill: auto and widows</title>
+
+  <link rel="author" title="Gérard Talbot" href="http://www.gtalbot.org/BrowserBugsSection/css21testsuite/">
+  <link rel="help" href="https://www.w3.org/TR/css-multicol-1/#filling-columns">
+  <link rel="help" href="https://www.w3.org/TR/css-break-3/#widows-orphans">
+  <link rel="match" href="reference/columnfill-auto-widows-002-ref.html">
+
+  <!--
+  Date created: November 7th 2020
+  Last modified: November 7th 2020
+  -->
+
+  <!--
+  Bug 1562420: Implement orphans and widows for individual columns of a multicolumn element
+  https://bugzilla.mozilla.org/show_bug.cgi?id=1562420
+  -->
+
+  <meta name="flags" content="">
+
+  <style>
+  div
+    {
+      border: orange solid 4px;
+      font-family: "Liberation Serif", "Times New Roman", serif;
```

Comment:
If all this test wants to test is orphans / widows support, I suggest that it be simplified, by removing all text and inserting explicit line breaks (BR elements) instead. That would be much easier to follow, and we won't have to worry about font rendering engine differences resulting in different line breaks.

Reply:
I have implemented your simplification suggestion into the 349b287 commit.